src/fights.ts

```diff
@@ -370,6 +373,11 @@ function embezzlerSetup() {
   if (!get("_cameraUsed") && !have($item`shaking 4-d camera`)) {
     retrieveItem($item`4-d camera`);
   }
+
+  // Fix invalid spooky putty monster (caused by ascending)
```

Comment:
You may want to examine the spooky putty monster here, to make sure mafia knows what's in there.

Reply:
yeah i'll add that